lib/common/tests/strings/pcmk__strcmp.c

```diff
@@ -0,0 +1,75 @@
+#include <glib.h>
+
+#include <crm_internal.h>
+#include <crm/common/strings_internal.h>
+
+static void
+same_pointer(void) {
+    const char *s1 = "abcd";
+    const char *s2 = "wxyz";
+
+    g_assert(pcmk__strcmp(s1, s1, pcmk__str_none) == 0);
+    g_assert(pcmk__str_eq(s1, s1, pcmk__str_none) == TRUE);
+    g_assert(pcmk__strcmp(s1, s2, pcmk__str_none) != 0);
+    g_assert(pcmk__str_eq(s1, s2, pcmk__str_none) == FALSE);
+    g_assert(pcmk__strcmp(NULL, NULL, pcmk__str_none) == 0);
```

Comment:
Very tiny nitpick, you defined pcmk__str_eq() as bool, so this should use true/false

Having been bit by that byte ordering thing earlier makes me a little more paranoid about these things :)

Reply:
Fixed.